### PR TITLE
[master] EMBL hwobj

### PR DESCRIPTION
**Pull request by IvarsKarpics**

### CODE REVIEW

File: HardwareObjects/EMBL/EMBLBeamlineTest.py [hunk 1]
```diff
@@ -1476,6 +1506,26 @@ def test_measure_intensity(self):
         self.emit("progressStop", ())
         return result
 
+    def test_file_system(self):
+        result = {}
+        result["result_bit"] = False
+        result["result_short"] = "Failed"
+        result["result_details"] = []
+
+        print self.bl_hwobj.session_hwobj.get_base_image_directory()
```
Comment (marcus-oscarsson) [inline HardwareObjects/EMBL/EMBLBeamlineTest.py]: Are you sure that you want these are prints and not log ?

**Comment by marcus-oscarsson** [comment]
Looks good to me, let me know if you intend to change the prints ...

**Comment by IvarsKarpics** [comment]
Thanks for a review and pointing to the prints. I removed them.

**Comment by marcus-oscarsson** [comment]
:)

**Comment by IvarsKarpics** [comment]
Thanks!

**Pull request merged**